src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/QueryParameterValueResolver.php

```diff
@@ -105,6 +113,10 @@ public function resolve(Request $request, ArgumentMetadata $argument): array
             $value = \is_array($value) ? array_map($enumFrom, $value) : $enumFrom($value);
         }
 
+        if (null !== $uidType) {
+            $value = $uidType::fromString($value);
```

Comment:
I think this misses support for variadics, isn't it? (aka when $value is an array of strings that should be turned into an array of uids)

Reply:
@seb-jean Can you have a look?

Reply:
I added support for variadics.